pkg/symlink/fs_test.go

```diff
@@ -400,3 +400,17 @@ func TestFollowSymlinkNoLexicalCleaning(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestFollowSymlinkWildcard(t *testing.T) {
```

Comment:
it is. Just to make sure that the symlink library doesn't try to expand glob matches and put them in right context.